src/main/app/claims/models/reviewOrder.ts

```diff
@@ -0,0 +1,29 @@
+import { Moment } from 'moment'
+import { MomentFactory } from 'shared/momentFactory'
+
+export class ReviewOrder {
+  reason: string
+  requestedBy: RequestedBy
+  requestedAt: Moment
+
+  constructor (reason?: string, requestedBy?: RequestedBy, requestedAt?: Moment) {
+    this.reason = reason
+    this.requestedBy = requestedBy
+    this.requestedAt = requestedAt
+  }
+
+  deserialize (input: any): ReviewOrder {
+    if (input) {
+      this.reason = input.reason
+      this.requestedBy = input.requestedBy
+      this.requestedAt = MomentFactory.parse(input.requestedAt)
+    }
+
+    return this
+  }
+}
+
+export enum RequestedBy {
```

Comment:
Please use the existing MadeBy class rather than creating a new one

Reply:
MadeBy is a form model. We generally keep form models separate from backend models. Happy to change if necessary.

Reply:
@andrewwa-kainos, let me know what you think here please?

Reply:
@KiranVarma02 good call that we shouldn't use form models in these cases, but actually it looks like that is the model we use for comms with the claim-store and the `madeBy.ts` file is just in an inappropriate place. 

@robertparkinson could we raise tech-debt to move `madeBy.ts` to `app/models`?

Reply:
using MadeBy now

Reply:
@KiranVarma02 can you please do the move as andy mentions above as part of this PR? other wise it will take too long to get it done